.github/workflows/pr-workflow-check.yaml

```diff
@@ -20,6 +20,13 @@ jobs:
       github_event_pull_request_head_repo_id : ${{ github.event.pull_request.head.repo.id }}
       github_workflow: ${{ github.workflow }}
       github_event_pull_request_head_sha: ${{ github.event.pull_request.head.sha }}
+      flow: |
+        ${{ 
+          (github.event_name == 'push' && 'push') ||
+          (inputs.github_event_name == 'merge_group' && 'merge_queue_check') ||
+          (inputs.github_event_name == 'pull_request_target' && inputs.github_event_pull_request_head_repo_id != 383289760 && 'pr_from_fork') ||  
```

Comment:
What’s this id?

Reply:
this is hack around GH. GH runs fork and branch PR flows at same time and also it does not allows to make green job if want to skip its deps (so skip steps work well, but need shortcut for it).  fixing GH action (likely reason people migrate from GH instead of fix btw).